configs/agent/halfstack.toml

```diff
@@ -15,6 +15,7 @@ pid-file = "./agent.pid"
 event-loop = "uvloop"
 var-base-path = "./var/lib/backend.ai"
 # allow-compute-plugins = []
+image-commit-path = "/tmp/backend.ai/commit/"
```

Comment:
If the default value is `/tmp`, it would break with Snap-based Docker installations. I'd recommend using a directory under the current working directory of agent.